cloud-run/fhir-converter/main.py

```diff
@@ -170,5 +172,12 @@ def convert_to_fhir(
         result = json.load(open(output_data_file_path))
     else:
         result = vars(converter_response)
+        # Include original input data in the result.
+        result["original_request"] = {
+            "input_data": input_data,
+            "input_type": input_type,
+            "filename": filename,
```

Comment:
Given that the original file name isn't required for conversion I don't think we should pass it to the converter. My thought is that the `read_source_data` function will provide the filename to the Workflow orchestration layer and in the event that there is a conversion failure the Workflow will include the file name in its all to the Cloud Function this PR is introducing. I think this approach also aligns better with our discussion yesterday to think about this converter being used more broadly. What are you thoughts?

Reply:
Ahh, I wasn't sure if the workflow would hang on to the filename provided by `read_source_data`. In that case, I agree with you, will remove!